Code/Tools/TestImpactFramework/Runtime/Code/Source/TestImpactRuntimeUtils.cpp

```diff
@@ -14,6 +14,8 @@
 #include <Artifact/Factory/TestImpactBuildTargetDescriptorFactory.h>
 #include <Artifact/Static/TestImpactTargetDescriptorCompiler.h>
 
+#include <AZCore/JSON/document.h>
```

Comment:
update: Looks like we can remove this include as nothing in this file has changed? Do we really need to add this?

Reply:
Will remove this.